app/controllers/sign_in/service_account_application_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SignIn
+  class ServiceAccountApplicationController < ActionController::API
+    include SignIn::Authentication
+    include SignIn::ServiceAccountAuthentication
+    include Pundit::Authorization
+    include ExceptionHandling
+    include Headers
+    include SentryLogging
+    include SentryControllerLogging
```

Comment:
I think you're going to have to `include Traceable` and add `service_tag 'identity'` now that this is a Platform requirement for controllers nowadays